### PR TITLE
Update the front end to use Material UI Dahsboard template

### PR DESCRIPTION
**Pull request by arcturus2**
* See https://github.com/mui-org/material-ui/tree/master/docs/src/pages/getting-started/templates/dashboard
* extracted the urls to a variable

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,59 +1,249 @@
...
+            height: 240,
+        },
+    }));
+
 function App() {
-  
-  const [tableViewToggled, setTableViewToggled] = useState(true);
-  const [tableViewSideToggled, setTableViewSideToggled] = useState(false);
+    const classes = useStyles();
+    const [open, setOpen] = React.useState(false);
```
Comment (antares1470) [inline src/App.js]: useState has been imported so you should be able to use it without prefixing with `React.`

**Review by antares1470 — CHANGES_REQUESTED** [review]

**Review by aldebaran30701 — APPROVED** [review]
Looks sleek! I would think it would be easier to manipulate the views if they were locked to the screen bounds rather than scrolling the whole page. (Views can still contain scrollbars) 

Just a thought though!

File: src/App.js (hunk 1 above)
Comment (arcturus2) [reply src/App.js]: Thanks for this feedback. Updated this and also some additional cleanup.

**Review by antares1470 — APPROVED** [review]

**Comment by arcturus2** [comment]
> Looks sleek! I would think it would be easier to manipulate the views if they were locked to the screen bounds rather than scrolling the whole page. (Views can still contain scrollbars)
> 
> Just a thought though!

Yeah @aldebaran30701 I do actually want to remove the scroll bars, one of the reasons I wanted to put this in a PR instead of overriding whats there. I am not sure how to do this at this stage - if you know that would be good to have fixed.

**Comment by arcturus2** [comment]
OK going to merge this in so it does not get out of sync with others working on the `develop` branch but noting the scroll bar issue needs to be fixed.

**Pull request merged**